tests/load/pipeline/test_clickhouse.py

```diff
@@ -1,20 +1,37 @@
-from typing import Iterator
+from typing import Iterator, Any, Generator
 
 import pytest
 
 import dlt
 from dlt.common.typing import TDataItem
 from dlt.common.utils import uniq_id
+from dlt.common.libs.pyarrow import row_tuples_to_arrow
+from dlt.common.destination import DestinationCapabilitiesContext
 from tests.load.utils import destinations_configs, DestinationTestConfiguration
-from tests.pipeline.utils import load_table_counts
+from tests.pipeline.utils import (
+    load_table_counts,
+    load_tables_to_dicts,
+    assert_load_info,
+    assert_records_as_set,
+)
+
+
+@pytest.fixture
+def pipeline(destination_config: DestinationTestConfiguration) -> Generator[Any, None, None]:
+    pipeline = destination_config.setup_pipeline(f"clickhouse_{uniq_id()}", dev_mode=True)
+    yield pipeline
+    with pipeline.sql_client() as client:
+        client.drop_dataset()
```

Comment:
why you need this? we already have a fixture that drops the dataset (also the staging one) of a pipeline. we run it after every test

Reply:
Yes, I just carried this over from the other test thinking that there's something special about ClickHouse. Will adjust.

Reply:
Fixed in b74deb790239b54fa53c9a02af9d9c981a591cd0